pjsip/src/pjsua-lib/pjsua_media.c

```diff
@@ -3467,6 +3467,10 @@ static void check_srtp_roc(pjsua_call *call,
             new_rem_addr = &new_si_->info.vid.rem_addr;
         }
 #endif
+        else {
```

Comment:
`check_srtp_roc()` is only called from `pjsua_media_channel_update()` when call_med->type is audio (surprisingly the video counterpart checking is missing! so this is actually the bug)

Reply:
I fixed the missing video SRTP roc checking in #3479